radosgw-agent/build/build

```diff
@@ -23,21 +44,22 @@ if [[ -f /etc/redhat-release || -f /usr/bin/zypper ]] ; then
             python setup.py bdist_rpm
             if [ $? -eq 0 ]
             then
-                mv dist/*.noarch.rpm $WORKSPACE/dist/noarch/.
-                mv dist/*.src.rpm $WORKSPACE/dist/SRPMS
+                find dist -name "*.noarch.rpm" | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/noarch
+                find dist -name "*.src.rpm" | $VENV/chacractl binary ${chacra_flags} create ${chacra_endpoint}/source
             fi
         else
+            mkdir -p $HOME/rpmbuild/{BUILD,BUILDROOT,RPMS,SOURCES,SPECS,SRPMS}
+
             python setup.py clean
             python setup.py sdist --formats=gztar
-            rpmdev-setuptree
-            rpmdev-wipetree
```

Comment:
Alright, I've removed this commit and added ``rpmdevtools`` to our ansible for jenkins slaves instead.

We'll want to make the ceph-deploy job do this as well.